### PR TITLE
Add maxWidth to selects

### PR DESCRIPTION
**Pull request by mdole**
Pretty much title. We want to have the ability to set maxWidth for some work on the /artworks page in Volt.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-palette* ready!

Built with commit a01b33566aafdfd38e6a05a012d01b375c02b5ef

https://deploy-preview-609--artsy-palette.netlify.com

File: packages/palette/src/elements/Select/Select.tsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React, { SFC } from "react"
 import styled, { css } from "styled-components"
 import {
+  maxWidth,
+  MaxWidthProps,
```
Comment (oxaudo) [inline packages/palette/src/elements/Select/Select.tsx]: I think we should have an option to set both `maxWidth` and `minWidth` here. And let clients figure out which one they want.

**Comment by oxaudo** [comment]
@mdole - also bunch of those chromatic changes seem unrelated. I do still scan and approve them usually just to make sure that nothing looks completely off.

There might be a better process with this though. Maybe @damassi or @zephraph can fill us in on the latest procedure with those.

File: packages/palette/src/elements/Select/Select.tsx (hunk 1 above)
Comment (mdole) [reply packages/palette/src/elements/Select/Select.tsx]: makes sense! done

**Comment by artsyit** [comment]
:rocket: PR was released in v5.1.1 :rocket:

**Pull request merged**